zuul.d/arista-eos-jobs.yaml

```diff
@@ -6,6 +6,9 @@
     parent: ansible-network-appliance-base
     pre-run: playbooks/ansible-network-eos-appliance/pre.yaml
     run: playbooks/ansible-network-eos-appliance/run.yaml
+    vars:
+      zuul_ssh_key_algorithm: ecdsa
+      zuul_ssh_key_size: 521
```

Comment:
We won't do this any more. The fix will be to update base-minimal-test to generate both rsa / ecdsa keys so both keys are on disk

Reply:
https://github.com/ansible/project-config/blob/master/playbooks/base-minimal-test/pre.yaml#L15

if you want to propose a PR to remove our fork, that is step 1